tests/test_minhash.py

```diff
@@ -1675,10 +1675,18 @@ def test_intersection_1_num():
     mh2.add_hash(2)
 
     mh3 = mh1.intersection(mh2)
-    print(set(mh3.hashes))
+    print("mh.intersection INTERSECTION HASHES:",set(mh3.hashes))
     assert len(mh3) == 1
     assert 0 in mh3.hashes
 
+def test_and_operator():
```

Comment:
whups, and `test_and_operator` also only works with empty minhashes (see comment below).